app/controllers/conference_controller.rb

```diff
@@ -5,7 +5,7 @@ class ConferenceController < ApplicationController
   load_resource :program, through: :conference, singleton: true, except: :index
 
   def index
-    @current = Conference.where('end_date >= ?', Date.current).order('start_date ASC')
+    @current = Conference.where('end_date >= ?', Date.current).reorder('start_date ASC')
```

Comment:
By default we order by `start_date DESC`.

https://github.com/openSUSE/osem/blob/master/app/models/conference.rb#L10

That's why. I guess you should change it there...